src/circadian_model.R

```diff
@@ -1,45 +1,85 @@
 # Here we will have all the functions related to the circadian model
 # We will be implementing Hannay's 2019 single population model plus
 # the sleep drive model, just like you have in your code
+library(deSolve)
+library(tidyverse)
 
 # Model parameters
+Dwake <-555.4
+Dsleep <- 572.7 
 # TODO: add parameters
-# alpha0 <- ...
-# p <- ...
-# I0 <- ...
-# ...
-# Define parameters vector
-# parameters <- c(...)
+alpha0 <- 0.05
+p <- 1.5
+I0 <- 9325.0
+Delta <- 0.0075
+G <- 33.75
+
+Irecep <- 1 / 291.0
+Targc <- 0.9677
+Tau <- 23.84
+Beta1 <- -0.09318
+A1    <- 0.3855
+A2    <- 0.1977
+BetaL1<- -0.0026 
+BetaL2<- -0.957756
+Sigma <- 0.0400692
+K     <- 0.06458
+Gamma <- 0.024
 
+# Define parameters vector
+sleepParameters <- c(Dwake, Dsleep)
+lightParameters <- c(alpha0, p, I0, Delta, G)
+circadianParameters <- c(Irecep, Targc, Tau, Beta1, A1, A2, BetaL1, BetaL2, Sigma, K, Gamma)
 # Model initial conditions
 # TODO: add initial conditions
 # ...
-# initial_conditions <- c(R=..., Psi=..., n=..., A=..., R1tot=...)
+initial_conditions <- c(R=0.8240484019863923, Psi=2.4532409247791804, n=0.78424752754260862, A=767.657, R1tot=584.24)
```

Comment:
Let's rename this `default_initial_conditions` so we know this is a value we provide.

Reply:
Alright, sounds good.